pkg/plugins/accesscontrol.go

```diff
@@ -26,5 +38,35 @@ func DeclareRBACRoles(acService ac.AccessControl) error {
 		},
 		Grants: []string{string(models.ROLE_VIEWER)},
 	}
-	return acService.DeclareFixedRoles(AppPluginsReader)
+	PluginsWriter := ac.RoleRegistration{
+		Role: ac.RoleDTO{
+			Name:        ac.FixedRolePrefix + "plugins:writer",
+			DisplayName: "Plugin Writer",
+			Description: "Enable and disable plugins and edit plugins' settings",
+			Group:       "Plugins",
+			Permissions: []ac.Permission{
+				{Action: ActionSettingsWrite, Scope: ScopeProvider.GetResourceAllScope()},
+			},
+		},
+		Grants: []string{string(models.ROLE_ADMIN)},
+	}
+	PluginsMaintainer := ac.RoleRegistration{
+		Role: ac.RoleDTO{
+			Name:        ac.FixedRolePrefix + "plugins:maintainer",
+			DisplayName: "Plugin Maintainer",
+			Description: "Install, uninstall plugins",
+			Group:       "Plugins",
+			Permissions: []ac.Permission{
+				{Action: ActionInstall},
+			},
+		},
+		Grants: []string{ac.RoleGrafanaAdmin},
+	}
+
+	// Remove possibility to install plugins if `plugin_admin_enable` is false
+	if !cfg.PluginAdminEnabled {
```

Comment:
If I'm not mistaken, we'll also need to verify if `|| cfg.PluginAdminExternalManageEnabled` here - a configuration we use in Cloud that brings users to grafana.com to manage their plugins instead